02-getting_started.Rmd

```diff
@@ -1,67 +1,135 @@
 
-2. R uses a ‘major.minor.patchlevel’ version numbering scheme. Accordingly there are three main releases of R available to install:
+[^02-getting_started-1]: Also known as [semantic versioning](https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning "Wikipedia explanation of semantic versioning")
 
-   * The official release (`r-release`), 
+There are three releases of R available to install:
 
-   * The patched release (`r-patched`), and
+-   The latest official release (`r-release`),
 
-   * The development release (`r-devel`). 
+-   The patched release (`r-patched`), and
 
-The `r-devel` is the next minor or eventually major release development version of R. Mostly, bug fixes are introduced in `r-patched`, while `r-devel` is for introducing new features.
+-   The development release (`r-devel`).
 
-## Installing R {#installR}
+The source code of released versions of R can be found at [R/tags](https://svn.r-project.org/R/tags/ "svn release source code folder"), the patched versions are at [R/branch](https://svn.r-project.org/R/branches/ "svn patched source code folder").
 
-1. The binary builds of R for Windows can be downloaded and installed from [here](https://cran.r-project.org/bin/windows/base/). Along with the link to the latest stable release, this page also contains links to the binary builds of r-patched and r-devel. 
+The `r-devel` at [R/trunk](trunk "svn devel source code folder") is the next minor or eventual major release development version of R.
+Mostly, bug fixes are introduced in `r-patched` and `r-devel`, while `r-devel` is for introducing new features.
 
-2. Click on the download links to download an executable installer.
+## Installing R in Linux
 
-3. Select the language while installing, read the GNU general public license information, and select the destination location to start the installation. You will be prompted to select components at this stage: `User installation`, `32-bit User installation`, `64-bit User installation`, or `Custom installation`. The default option may be chosen for the questions from this step onwards to complete the installation. 
+Here are the basic steps needed meant as a checklist.For complete instructions please see the section in [R-admin](https://cran.r-project.org/doc/manuals/r-devel/R-admin.html#Installing-R-under-Unix_002dalikes)
 
-## Building R and R packages
+1.  Retrieve R source code via into `TOP_SRCDIR`, note that we retrieve the r-devel source code:
 
-### What tools do you need to build R from source on Windows?
+        svn checkout https://svn.r-project.org/R/trunk/ TOP_SRCDIR
 
-1. [RTools](https://github.com/r-windows/docs/blob/master/faq.md#what-is-rtools) is the toolchain bundle that you can use to build R base and R packages containing compiled code, on Windows.
+    If you need to install svn you can use your distribution's package manager to install it.
```

Comment:
Yes, sorry for not being clear. I was referring to the part beginning "If you need to install svn you can use your distribution's package manager to install it." (I forgot I could highlight multiple lines!). 

I mean that you should say earlier that the instructions get the sources from the svn repo and the reader will need Subversion installed, then give the command for this on Linux. So you might make this part a "Step 0" in the instructions, or have a short "Prerequisites" sub-section before a "Building R" section.

Reply:
Will do it. I like more the Prerequisites subsection as it might be a good place to name the R dependencies: ubuntu packages, macos packages and windows programs.